README.md

```diff
@@ -180,7 +191,7 @@ use JMS\Serializer\Annotation as Serializer;
 use Hateoas\Configuration\Annotation as Hateoas;
 
 #[Serializer\XmlRoot('user')]
-#[Hateoas\Relation('self', href: "expr('/api/users/' ~ object.getId())")]
+#[Hateoas\Relation(name: 'self', href: "expr('/api/users/' ~ object.getId())")]
```

Comment:
is the `name` argument needed? it seems that before, it was optional, making it look like annotations

Reply:
Thank you for the hint.

It is not necessary anymore after 9e8a232bc00d9d97269712c33b67f088a87211e4